docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,339 @@
+	the left and make sure you select _Boot from image_, so you can choose 
+	one of the readily available OS images to boot the new server off of. 
+	To pick one of the images, click on the dropdown menu on the right. For 
+	this how-to guide, we have chosen _ubuntu_ in general and 
+	_Ubuntu 22.04 Jammy Jellyfish 20220810_ in particular.
+		
+	![Boot source](assets/new-server/shot-03.png)
+		
+	Next, make sure _Boot Target_ is set to _Volume (Recommended)_. 
+	Regarding the server's CPU core count and amount of memory, set the 
+	[_Flavor_](/reference/flavors) accordingly.
+	The default flavor specifies a server with 1 CPU core and 1GB of RAM.
+	You can start with that or select a different configuration by clicking
+	the dropdown menu at the right of _Flavor_. Please note that, depending on 
+	the chosen flavor, the estimated monthly cost of the server changes. (While a 
+	server is shut off you are still getting charged for it, but less so.) 
+	At any time, this estimated cost is displayed in the green rectangular area 
+	at the top. Something else that affects the cost is the size of the root 
+	device. Take a look at the _Size_ parameter below, and notice the 
+	default (in 
+	[gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary))). 
+	You may leave the root device size unchanged, or modify it to be lower 
+	or higher than the default.
+		
+	When, at a later time, you decide to delete the server, you can 
+	do so but **keep** its boot volume (you may want, for example, to 
+	attach the exact same volume to a new server). Just leave the _Delete 
+	on termination_ option disabled if you want this kind of flexibility. 
+	On the other hand, if you want your root volume to be automatically
+	deleted when the server terminates, enable _Delete on termination_.
+	Use this option with caution.
+		
+	Finally, you may enable _Disaster recovery_ for the server. If 
+	you do, then daily server snapshots will be created, and you will have 
+	the option for easy and fast rollups to previous snapshots. Please be 
+	aware that enabling this option increases the server's monthly estimated
+	cost (again, it is displayed in the green rectangular area at the top).
+		
+	![Server configuration](assets/new-server/shot-04.png)
+		
+	Regarding networking, select one of the available networks to 
+	attach the new server to. If you want the server accessible from the 
+	Internet, do not forget to enable the _I want an external IP for my 
+	server_ option. In the section below, set _Security Groups_ to 
+	_default_.
+		
+	![Networking and security groups](assets/new-server/shot-05.png)
+		
+	If you already have one or more public keys in your 
+	{{extra.brand}} Cloud account, you can now select a key to be included 
+	in the `~/.ssh/authorized_keys` file of the server's default user. That 
+	way, you can securely log into the remote user's account without typing 
+	a password. If there are no public keys to choose from, activate the 
```

Comment:
I think we should actively discourage password logins, since they are eminently less safe than keypair based ones. So I think we should, as soon as we have a how-to for uploading public keys, replace this paragraph with a link to that how-to.

Reply:
Totally agree with that. As soon as the new how-to is up this paragraph is gone -- and the whole how-to is updated wherever needed.